doc/rofi.1.markdown

```diff
@@ -585,6 +585,9 @@ Default: *"wmctrl -i -R {window}"*
 
 Show window thumbnail (if available) as icon in the window switcher.
 
+`-window-prefer-icon-theme`
+
+Use icons from the current GTK theme in window mode.
```

Comment:
It is not 'gtk theme'.  It is the icon theme set, this is not directly dependent on gtk. 
Rofi f.e. has it own set icon-theme option, and this will be used when set.

Reply:
I changed it to  'icon theme', does it need to be more verbose in some other way?